cumulus/tasks/copy-idx-from-s3-to-efs/package.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
could you add the key `"private": true` to the package.json to avoid publishing them to npm (at least not yet)